### PR TITLE
Automatically port System.Xml.Linq.XDocument triple slash

### PR DESCRIPTION
**Pull request by carlossanlop**
Area owners @buyaa-n @krwq

### CODE REVIEW

File: xml/System.Xml.Linq/XDocument.xml [hunk 1]
```diff
@@ -1654,12 +1654,20 @@ Root                1    2
...
-        <param name="stream">To be added.</param>
-        <param name="options">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="stream">A <see cref="T:System.IO.Stream" /> containing the raw XML to read into the newly created <see cref="T:System.Xml.Linq.XDocument" />.</param>
+        <param name="options">A set of <see cref="T:System.Xml.Linq.LoadOptions" />.</param>
+        <param name="cancellationToken">A cancellation token.</param>
+        <summary>Create a new <see cref="T:System.Xml.Linq.XDocument" /> and initialize its underlying XML tree using the passed <see cref="T:System.IO.Stream" /> parameter.  Optionally whitespace handling can be preserved.</summary>
```
[the remark this answers is not in the document]
Comment (rpetrusha) [reply xml/System.Xml.Linq/XDocument.xml]: I've added it, @mairaw.

[the remark this answers is not in the document]
Comment (carlossanlop) [reply xml/System.Xml.Linq/XDocument.xml]: Thanks for catching it, @krwq. If you can, please use GitHub's suggestion feature, it's easier and faster to apply the changes with that feature.

[the remark this answers is not in the document]
Comment (carlossanlop) [reply xml/System.Xml.Linq/XDocument.xml]:
```suggestion
        <summary>Create a new <see cref="T:System.Xml.Linq.XDocument" /> and initialize its underlying XML tree using the passed <see cref="T:System.IO.Stream" /> parameter. Optionally whitespace handling can be preserved with the <xref:System.Xml.Linq.LoadOptions.PreserveWhitespace> setting.</summary>
```

File: xml/System.Xml.Linq/XDocument.xml [hunk 2]
```diff
@@ -2842,11 +2858,11 @@ Console.WriteLine(File.ReadAllText("Root2.xml"))
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="2" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="textWriter">To be added.</param>
-        <param name="options">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="textWriter">The <see cref="T:System.IO.TextWriter" /> to output the XML to.</param>
+        <param name="options">If <see cref="F:System.Xml.Linq.SaveOptions.DisableFormatting" /> is enabled the output is not indented. If"<see cref="F:System.Xml.Linq.SaveOptions.OmitDuplicateNamespaces" /> is enabled duplicate namespace declarations will be removed.</param>
```
[the remark this answers is not in the document]
Comment (carlossanlop) [reply xml/System.Xml.Linq/XDocument.xml]:
```suggestion
        <param name="options">A set of load options.</param>
```

File: xml/System.Xml.Linq/XDocument.xml [hunk 3]
```diff
@@ -2842,11 +2863,11 @@ Console.WriteLine(File.ReadAllText("Root2.xml"))
...
-        <param name="options">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="textWriter">The text writer to output the XML to.</param>
+        <param name="options">If <see cref="F:System.Xml.Linq.SaveOptions.DisableFormatting" /> is enabled the output is not indented. If"<see cref="F:System.Xml.Linq.SaveOptions.OmitDuplicateNamespaces" /> is enabled duplicate namespace declarations will be removed.</param>
+        <param name="cancellationToken">A cancellation token.</param>
+        <summary>Writes this <see cref="T:System.Xml.Linq.XDocument" /> to a <see cref="T:System.IO.TextWriter" />.</summary>
+        <returns>A task representing the asynchronous save operation.</returns>
         <remarks>To be added.</remarks>
```
Comment (carlossanlop) [inline xml/System.Xml.Linq/XDocument.xml]:
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

If <xref:System.Xml.Linq.SaveOptions.DisableFormatting?displayProperty=nameWithType> is set, the output is not indented. If <xref:System.Xml.Linq.SaveOptions.OmitDuplicateNamespaces?displayProperty=nameWithType> is set, duplicate namespace declarations will be removed.

          ]]></format>
        </remarks>
```

File: xml/System.Xml.Linq/XDocument.xml [hunk 4]
```diff
@@ -2806,12 +2822,17 @@ Console.WriteLine(File.ReadAllText("Root2.xml"))
...
+        <param name="options">A set of load options.</param>
+        <param name="cancellationToken">A cancellation token.</param>
+        <summary>Output this <see cref="T:System.Xml.Linq.XDocument" /> to a <see cref="T:System.IO.Stream" />.</summary>
+        <returns>A task representing the asynchronous save operation.</returns>
+        <remarks>  
+          <format type="text/markdown"><![CDATA[  
+
+If <xref:System.Xml.Linq.SaveOptions.DisableFormatting?displayProperty=nameWithType> is set, the output is not indented. If <xref:System.Xml.Linq.SaveOptions.OmitDuplicateNamespaces?displayProperty=nameWithType> is set, duplicate namespace declarations will be removed.
+
+         ]]></format>
```
Comment (carlossanlop) [inline xml/System.Xml.Linq/XDocument.xml]:
```suggestion
         ]]></format>
       </remarks>
```

File: xml/System.Xml.Linq/XDocument.xml [hunk 5]
```diff
@@ -2806,12 +2822,17 @@ Console.WriteLine(File.ReadAllText("Root2.xml"))
...
-        <remarks>To be added.</remarks>
+        <param name="stream">The stream to write the XML to.</param>
+        <param name="options">A set of load options.</param>
+        <param name="cancellationToken">A cancellation token.</param>
+        <summary>Output this <see cref="T:System.Xml.Linq.XDocument" /> to a <see cref="T:System.IO.Stream" />.</summary>
+        <returns>A task representing the asynchronous save operation.</returns>
+        <remarks>  
+          <format type="text/markdown"><![CDATA[  
+
+If <xref:System.Xml.Linq.SaveOptions.DisableFormatting?displayProperty=nameWithType> is set, the output is not indented. If <xref:System.Xml.Linq.SaveOptions.OmitDuplicateNamespaces?displayProperty=nameWithType> is set, duplicate namespace declarations will be removed.
```
Comment (carlossanlop) [inline xml/System.Xml.Linq/XDocument.xml]:
```suggestion
## Remarks

If <xref:System.Xml.Linq.SaveOptions.DisableFormatting?displayProperty=nameWithType> is set, the output is not indented. If <xref:System.Xml.Linq.SaveOptions.OmitDuplicateNamespaces?displayProperty=nameWithType> is set, duplicate namespace declarations will be removed.
```

**Comment by carlossanlop** [comment]
@mairaw @rpetrusha all comments addressed, build passed without warnings. Is this good to merge?

**Comment by rpetrusha** [comment]
Yes, it's good to merge, @carlossanlop. I'll do that now.

**Pull request merged**